### ISSUE TITLE
Feature: Color coding by topic

### ISSUE THREAD

**Issue by yousraHazem** [quran/quran_android#684]
Suggestion to add a feature of color-coding the Musshaf by topic as an option in the settings.

**Comment by ahmedre** [comment]
jazakumAllah khairan - can you elaborate some more? you want certain ayat color coded to signify they are talking about a certain topic?

**Comment by yousraHazem** [comment]
Correct, something like [this](https://photos.app.goo.gl/Fm4BZerXwoGrbJ3C7), where color signifies the topic; Traditionally more serious topics are color-coded in red, ayat about jannah are in green,... with alternating colors for adjacent topics. 

I have some time on my hands these days, so please let me know if PRs are welcome for this one.

**Comment by ahmedre** [comment]
would love to have something like this! do you know if there's a data set somewhere we can use, or would we have to make our own?

**Comment by yousraHazem** [comment]
so after some searching I think the dataset is not posted anywhere. I found many pdf's of the Musshaf with (التفصيل الموضوعي), but I think it would be tough to extract information from there, unless you're aware of some way we can extract it.  

On the other hand, I found many datasets with the reverse of what we want, providing a list of topics and relating them to different ayahs in the quran, so the relationship between the ayahs and the topics is a many to many relationship. 

I think we should explore ways to collect that data in the meantime, and I'm sure many people would be up to help with this.

 Regarding the format of the dataset we are looking at something like a csv as follows?
```
start_verse, end_verse, topic
1:1, 1:4, استحقاق الحمد للخالق وحده
1:5, 1:6, حصر العبادة لله وحده....
```

**Comment by Ahmadreza128** [comment]
Assalaamu’alaikum wa rahmatullah wabarakatuhu
I think this is the thing that you want
walsalam 3alaikum.
[موضوع ایات.xlsx](https://github.com/quran/quran_android/files/6511155/default.xlsx)
